options.html

```diff
@@ -36,6 +36,16 @@
       <label for="disableEverywhere" data-i18n="options_disable_extension_everywhere">Everywhere</label><br />
       <br />
 
+      <p data-i18n="options_work_mode">Work Mode</p>
```

Comment:
consider: Adding the strings to the `en` and `es` translation files in `_locales`. This makes things easier to translate in future.